client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfigurations.java

```diff
@@ -163,6 +168,19 @@ private static ClientConfiguration of(
                 .build();
     }
 
+    private static ImmutableSet<String> parseNoProxyEnvironmentVar() {
+        String env = System.getenv(ENV_NO_PROXY);
+        if (env == null || env.isEmpty()) {
+            log.debug(ENV_NO_PROXY + " environment variable not set.");
```

Comment:
To match the other `FROM_ENVIRONMENT` logic, I would set this to 
```suggestion
            log.info(ENV_NO_PROXY + " environment variable not set.");
```